scipy/linalg/decomp.py

```diff
@@ -1267,3 +1267,138 @@ def hessenberg(a, calc_q=False, overwrite_a=False, check_finite=True):
     q, info = orghr(a=hq, tau=tau, lo=lo, hi=hi, lwork=lwork, overwrite_a=1)
     _check_info(info, 'orghr (hessenberg)', positive=False)
     return h, q
+
+
+def cdf2rdf(w, v):
+    """
+    Converts complex eigensystems to real eigenvalues in a block diagonal form
+    and the associated real eigenvectors.
+
+    If the eigensystem:
+
+    `w, v = scipy.linalg.eig(X)`
+    or
+    `w, v = numpy.linalg.eig(X)`
+
+    has complex eigenvalues appearing in complex-conjugate pairs, cdf2rdf
+    transforms the system so `w` is in real diagonal form, with 2-by-2 real
```

Comment:
For inline codeblocks you need double backticks. Otherwise it tries to create a link to the functions.